### ISSUE TITLE
🐛 [CLI] biome.exe blocked by Windows Security after installing via winget install biomejs.biome

### ISSUE THREAD

**Issue by djwinston** [biomejs/biome#10505]
### Environment information

<details>
OS: Windows 11 Pro 25H2 (10.0.26200.8524)

Installation method: winget install biomejs.biome
</details>

### What happened?

1. Run `winget install biomejs.biome` in PowerShell
2. Attempt to run `biome --version` in terminal
3. Windows Security (Defender) blocks biome.exe from executing — the binary is flagged/quarantined on first run

Note: `biome rage` output is unavailable because the binary cannot be executed due to the block.
Previously encountered a similar issue with the VSCode extension's biome.exe, which was blocked by an ASR (Attack Surface Reduction) rule and required a manual exclusion via `Add-MpPreference -AttackSurfaceReductionOnlyExclusions`.

<img width="373" height="270" alt="Image" src="https://github.com/user-attachments/assets/57d4ec75-f7c9-4684-bf50-da37d8936bba" />

### Expected result

biome.exe installed via winget should be executable without being blocked by Windows Security or Windows Defender SmartScreen.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
We don't own the distribution of the binary on Winget, you'll have to reach out to the maintainer

**Comment by djwinston** [comment]
ℹ️ 
```powershell
Add-MpPreference -AttackSurfaceReductionOnlyExclusions $env:LOCALAPPDATA\Microsoft\WinGet\Packages\biomejs.biome*\biome.exe"
```
